README.md

```diff
@@ -40,6 +40,12 @@ In order to build Mozart 2, you need the following tools on your computer:
 *   Tcl/Tk 8.5 or 8.6 (with development files)
 *   emacs
 
+For building CLANG/LLVM:
+*   LibXML2-dev (tested with version 2.9.3)
+*   OCaml-findlib
+*   libCTypes-OCaml-dev (>= 0.4 - available in Debian Unstable as of Jan. 2016)
```

Comment:
I will need some time to look into this. I never installed these packages myself and it works, so it seems strange to me.
In the meantime, would you mind [squashing your commits into one](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) ?
There are six commits for 15 lines changed.